doc/thirdpartypackages/index.rst

```diff
@@ -134,6 +134,15 @@ toolkit written by Joe Kington to provide interactive "data cursors"
 (clickable annotation boxes) for matplotlib.
 
 
+.. _toolkit_mplcursors:
+
+mplcursors
+==========
+
+`mplcursors <https://github.com/anntzer/mplcursors>` provides interactive
```

Comment:
The trailing underscores are missing to turn this into a link. Also, why not link to RTD instead of GitHub?